### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by mshahbazi**
handle function arguments default value

### CODE REVIEW

**Comment by djrobstep** [comment]
Hey, thanks so much for this pull request! It looks good. Makes sense to use the inbuilt definition now that postgres provides it.

This breaks a couple of the tests as the inbuilt function definition output formats things slightly differently. Just need to fix that up and we can get this merged.

**Comment by mshahbazi** [comment]
Thanks you. This applies to the other definitions, e.g., tables, views, etc.

**Comment by djrobstep** [comment]
Are you happy to fix up the tests here, or would you rather I do that?

**Comment by mshahbazi** [comment]
I would be happy to be of any help. Please let me know if fixing issues in a new branch works.

**Comment by djrobstep** [comment]
@mshahbazi Fixing them in this branch should work OK. If you need any guidance just ask

**Comment by mshahbazi** [comment]
@djrobstep it seems I don't have write access to the repository.

**Comment by djrobstep** [comment]
@mshahbazi i meant just as part of this same pull request, in your own fork, if that makes sense?

**Comment by mshahbazi** [comment]
done. sorry for misunderstanding.

File: tests/test_schemainspect.py [hunk 1]
```diff
@@ -421,7 +422,7 @@ def n(name, schema="public"):
     assert t.alter_table_statement("x") == "alter table {} x;".format(t_films)
     assert n("films_title_idx") in t.indexes
     g = InspectedPrivilege("table", "public", "films", "select", "postgres")
-    g = i.privileges[g.key]
+    # g = i.privileges[g.key]
```
Comment (djrobstep) [inline tests/test_schemainspect.py]: Looks like you might have commented this out accidentally?

**Comment by djrobstep** [comment]
Looks great! Just one line you might have commented out accidentally. Correct that and we should be good to merge.

**Comment by djrobstep** [comment]
Merged now. Thanks so much for contributing this! There will be a new release soon including this.

**Comment by mshahbazi** [comment]
Thank YOU man! Your work saves a lot of time for many developers, including me.

**Pull request merged**